tomviz/Pipeline.h

```diff
@@ -81,6 +81,11 @@ protected slots:
   /// This signal is fired the execution of the pipeline finishes.
   void finished();
 
+  /// This signal is fired when an operator is added.  The second argument
+  /// is the datasource that should be moved to become its output in the pipeline
+  /// view (or null if there isn't one).
+  void operatorAdded(Operator* op, DataSource* outputDS);
```

Comment:
Can we just default outputDS to ```nullptr```, then we don't have to pass ```nullptr``` at 273 and 276.

Reply:
I wasn't sure since it is a signal, but apparently it works.

Reply:
Signals are just member functions, and so have all the same features available as member functions in terms of default arguments, etc.